protocol/communities/community_changes.go

```diff
@@ -268,3 +280,19 @@ func evaluateCommunityChangesByDescription(origin, modified *protobuf.CommunityD
 
 	return changes
 }
+
+func findDiffInArrays(searchFrom []string, searchIn []string, storeTo map[string]bool) {
+	for _, memberToFind := range searchFrom {
+		exists := false
+		for _, member := range searchIn {
```

Comment:
na: can be replaced with `strings.Contains`

Reply:
it does not exist in go (at least in my 1.18 version)

Reply:
I got confused, I meant `golang.org/x/exp/slices` with `slices.Contains`